homeassistant/components/airvisual/__init__.py

```diff
@@ -103,53 +113,75 @@ async def async_setup(hass, config):
     return True
 
 
-async def async_setup_entry(hass, config_entry):
-    """Set up AirVisual as config entry."""
+@callback
+def _standardize_geography_config_entry(hass, config_entry):
+    """Ensure that geography observables have appropriate properties."""
     entry_updates = {}
+
     if not config_entry.unique_id:
         # If the config entry doesn't already have a unique ID, set one:
         entry_updates["unique_id"] = config_entry.data[CONF_API_KEY]
     if not config_entry.options:
         # If the config entry doesn't already have any options set, set defaults:
-        entry_updates["options"] = DEFAULT_OPTIONS
+        entry_updates["options"] = {CONF_SHOW_ON_MAP: True}
+
+    if not entry_updates:
+        return
+
+    hass.config_entries.async_update_entry(config_entry, **entry_updates)
```

Comment:
Should we not keep any of the existing entry data, besides api key ?

Reply:
Can you give me more context on this one? Not understanding.

Reply:
We're replacing the entry data with `entry_updates`, so only that data will remain. We don't want to keep any other data that we originally had in the config entry?